common/script/aria.js

```diff
@@ -203,6 +194,28 @@ function ariaAttributeReferences() {
             node.replaceChild(l, parentNode);
           }
         }
+        // Annotate global attribute references
```

Comment:
This file is maintained in aria-common - but having it here is great for now as it lets us test the changes.

Reply:
Ah, I see, thanks.  Once this PR is finalized, would it be appropriate for me to create a separate PR [for this file in the `aria-common` repo](https://github.com/w3c/aria-common/blob/main/script/aria.js)?